### PR TITLE
Content Tree: Borderless children items

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
## Description

- All children items are borderless
- The last child decided by the user is set to have border-bottom and other sides are borderless

References #
#2193 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## How Has This Been Tested?

https://deploy-preview-2430--moduswebcomponents.netlify.app/?path=/story/components-content-tree--default

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by coliff — DISMISSED** [review]
Looks much better! 👍

**Comment by ElishaSamPeterPrabhu** [comment]
Can we merge this @coliff ?

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx [hunk 1]
```diff
@@ -331,12 +339,12 @@ const ActionBarTemplate = ({
...
       </modus-tree-view-item>
       <modus-tree-view-item node-Id="6" label="Archived">
-        <modus-tree-view-item node-Id="7" label="Folder1">
+        <modus-tree-view-item node-Id="7"  is-last-child=${isLastChild}  label="Folder1">
           <modus-tree-view-item
             node-Id="8"
             label="File1"></modus-tree-view-item>
           <modus-tree-view-item
-            node-Id="9"
+            node-Id="9"  is-last-child=${isLastChild}
```
Comment (coliff) [inline stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx:347]: looks like two spaces here - Prettier should fix this ideally.

**Comment by coliff** [comment]
> Can we merge this @coliff ?

Yep, ok. It looks better and will be good to get merged for the release tomorrow. I was wondering if the  last-child border bottom could be handled automatically using just CSS and nothing else. Could make the code cleaner if possible?

**Comment by ElishaSamPeterPrabhu** [comment]
> > Can we merge this @coliff ?
> 
> Yep, ok. It looks better and will be good to get merged for the release tomorrow. I was wondering if the last-child border bottom could be handled automatically using just CSS and nothing else. Could make the code cleaner if possible?

With slotting i was not able to acees the last child in css hence this approach

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx]: Prettier doesn't work in story book formatted manually

**Review by coliff — APPROVED** [review]

**Pull request merged**